PythonTool/PlotContour.py

```diff
@@ -0,0 +1,31 @@
+from visit import *
+
+
+def PlotContour(File, ObjectSequence):
+    # Contour plot attributes.
+
+    # Change orientation of legend
+    LegendAttributes = GetAnnotationObject(ObjectSequence["Contour"])
+    LegendAttributes.orientation = "VerticalLeft"
+    LegendAttributes.fontHeight = 0.03
+
+    Attribute = ContourAttributes()
+
+    Attribute.legendFlag = 1
+
+    if File is not None:
```

Comment:
This may be true, but the process for setting the plot attributes will likely remain the same. I'd still consider factoring this out into somekind of `generatePlotAttributes` function.